test/images/scripts/debian-unstable.setup

```diff
@@ -76,6 +78,10 @@ curl \
 useradd -m -U -c Administrator -G sudo -s /bin/bash admin
 echo admin:foobar | chpasswd
 
+# For storaged
+echo "deb http://ftp.uk.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
```

Comment:
https://wiki.debian.org/DebianExperimental#To_install_an_experimental_package recommends `deb http://httpredir.debian.org/debian experimental main` and let that pick a mirror
This change won't need an image respin.